deploy_s3watcher.sh

```diff
@@ -0,0 +1,24 @@
+#/bin/bash
```

Comment:
Not urgent, but would be lovely to have a standardised way to push a build to S3. Would be useful to deploy kahuna as static files, or other things like the wat pages and grid-inspector. I think there may be an sbt plugin that does this but obviously a lot of these things are *not* sbt projects, so would have to be more generic.

Just a thought.